package.json

```diff
@@ -193,6 +194,9 @@
             "ws@>=8.0.0 <8.17.1": "8.17.1",
             "ws@>=7.0.0 <7.5.10": "7.5.10"
         },
+        "patchedDependencies": {
```

Comment:
Why is there a patch needed? Should we open a PR to the heplers instead?

Reply:
I did this as `helpers` package does not expose `convertLegacyIdl` helper. That is done to check these helpers could not parse custom IDLs for some Anchor versions. For ones we use modified `convertLegacyIdl` implementation to display them properly in human-readable format.

As the version for this package is locked, that will hardly produce errros in future.

Reply:
I was just thinking if it would make sense to improve the function for everybody if you improved it. We can still do that in the next pr though